contracts/HubPool.sol

```diff
@@ -207,7 +207,8 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
         address originToken,
         address destinationToken
     ) public onlyOwner {
-        // TODO: Should relay message to L2 for destinationChainId and call setEnableRoute(originToken, destinationChainId, true)
+        // Note that this method makes no L1->L1 call to whitelist the route. The assumption is that the origin chain's
+        // SpokePool Owner will call setEnableRoute to enable the route. This removes the need for an L1->L2 call.
         whitelistedRoutes[originToken][destinationChainId] = destinationToken;
```

Comment:
This comment no longer applies as my outstanding PR #33 will make the L1-->L2 call

Reply:
yup. I saw that :)

Reply:
Can remove this comment